_includes/pbc/all/install-features/202304.0/install-the-customer-access-feature.md

```diff
@@ -0,0 +1,437 @@
+
```

Comment:
please create the corresponding page under docs/pbc/all/{PBC_NAME}/{VERSION}/install-and-upgrade as for 202212.0 release